OrbitQt/orbitmainwindow.cpp

```diff
@@ -630,8 +630,8 @@ void OrbitMainWindow::OnTimer() {
   ORBIT_SCOPE("OrbitMainWindow::OnTimer");
   app_->MainTick();
 
-  for (OrbitGLWidget* glWidget : m_GlWidgets) {
-    glWidget->update();
+  for (OrbitGLWidget* gl_widget : gl_widgets_) {
+    gl_widget->update();
```

Comment:
is the order important here?

Reply:
Good question. Probably not. In our current use case only one widget will be registered anyway - the capture window.

This should probably be refactored. Instead of giving OrbitGLWidget a pointer to OrbitMainWindow for calling RegisterGLWidget, OrbitMainWindow should maintain this list by itself. But I felt that this is out of scope of this PR.